lib/Applify.pm

```diff
@@ -170,8 +173,8 @@ OPTION:
 
     printf(
       " %s %2s%-${width}s  %s\n",
-      $option->{required} ? '*'  : $option->{n_of} ? '+' : ' ',
-      length($name) > 1   ? '--' : '-',
+      $option->{required} ? '*' : $option->{n_of} ? '+' : ' ',
+      length($name) > 1 ? '--' : '-',
       $name, $option->{documentation},
```

Comment:
Can you add the sorting of options?

Reply:
What do you mean? The options are listed as you specify in your code.

Reply:
Someone commented about sorting options alphabetically recently. I just noticed they don't get sorted.

Reply:
I'm not going to do that. If someone wants them sorted alphabetically, then they can easily do that inside their code.